shared/chat/conversation/create-team-header/container.js

```diff
@@ -14,12 +14,14 @@ const mapStateToProps = (state, {conversationIDKey}: OwnProps) => ({
 const mapDispatchToProps = dispatch => ({
   _onShowNewTeamDialog: (conversationIDKey: Types.ConversationIDKey) => {
     dispatch(
-      RouteTreeGen.createNavigateAppend({path: [
-        {
-          props: {conversationIDKey},
-          selected: 'showNewTeamDialog',
-        },
-      ]})
+      RouteTreeGen.createNavigateAppend({
+        path: [
+          {
+            props: {conversationIDKey},
+            selected: 'chatShowNewTeamDialog',
```

Comment:
This doesn't seem to work for me. from the info panel in chat and clicking turn this into a team

Reply:
yeah it doens't work

Reply:
there are lots of known issues that i didn't fix. this PR is to get this in (feature flagged) so we can work on sections as a team